object-cache.php

```diff
@@ -626,6 +664,16 @@ function set( $id, $data, $group = 'default', $expire = 0 ) {
 		return $result;
 	}
 
+	public function set_multiple( array $data, $group = '', $expire = 0 ) {
+		$values = array();
+
+		foreach ( $data as $key => $value ) {
+			$values[ $key ] = $this->set( $key, $value, $group, $expire );
```

Comment:
Can't you use this - https://www.php.net/manual/en/memcached.setmulti.php

Reply:
No, we use `Memcache` extension, not `Memcached`.

https://github.com/Automattic/wp-memcached/blob/master/object-cache.php#L849

Reply:
My bad. Never mind.